mods/ws_core/nodes.lua

```diff
@@ -791,6 +791,8 @@ minetest.register_node("ws_core:stalactites", {
 		ws_core.dig_up(pos, node, digger)
 	end,
 })
+<<<<<<< HEAD

```

Comment:
remove these lines (this and 5 similar ones), they're not automaically removed by git sometimes

Reply:
Found it only in two files?